### PR TITLE
[WIP] feat: help users use rclip with image viewers on Windows

### PR DESCRIPTION
**Pull request by TheChaoticor**
solved,connected with xnview

### CODE REVIEW

**Comment by TheChaoticor** [comment]
Please review it

File: rclip/main.py [hunk 1]
```diff
@@ -216,7 +219,7 @@ def main():
     args.exclude_dir,
     args.no_indexing,
   )
-
+  XNVIEW_PATH = "C:\\Program Files\\XnViewMP\\xnviewmp.exe"
```
Comment (yurijmikhalevich) [inline rclip/main.py:222]: 1. What if I have xnview installed to a different path?
2. What if I don't have xnview installed?

File: rclip/main.py [hunk 2]
```diff
@@ -228,6 +231,13 @@ def main():
         print(f'{r.score:.3f}\t"{r.filepath}"')
         if args.preview:
           preview(r.filepath, args.preview_height)
+        if os.path.exists(XNVIEW_PATH):
```
Comment (yurijmikhalevich) [inline rclip/main.py:234]: Why does this always try to open the xnview?

1. What if I am on Linux/macOS?
2. What if I don't want to open xnview?

File: rclip/main.py (hunk 1 above)
Comment (yurijmikhalevich) [reply rclip/main.py:222]: 3. Why xnview? What about other image viewers.

**Review by yurijmikhalevich — CHANGES_REQUESTED** [review]
Thank you for the initiative 🙌 

But the design of this PR is bad:
1. We probably shouldn't require the user to have xnview installed to use previews on Windows.
2. We shouldn't hardcode the path to xnview.
3. We shouldn't open result images in xnview by default.

A better version of this PR could be:
- no changes to rclip code
- a change to README.md showing how to use xnview together with rclip to view files

Another option may involve building a generic adapter/helper that may try calling an image viewer if the user passed a CLI arg to ask for this, but this is more involved, and I am not sure if we need to do it now. But happy to discuss different approach ideas.

Can you change this PR to:
- remove changes to the rclip code
- add an example on how to use rclip with xnview, similar to this one: https://github.com/yurijmikhalevich/rclip/blob/01801ea4d027804ab945ebf56902954544e7b3a8/README.md?plain=1#L121-L137

Thank you, and let me know if you have any questions 😄

**Comment by TheChaoticor** [comment]
Thank you for your review.I was inspired by the issue and suggestion on xnview and i misunderstood thinking that the pr can be made successful by just connecting with xnview .I will improve my pr and let you know

**Comment by yurijmikhalevich** [comment]
Closing as the PR is stale and broken.

**Pull request closed without merging**